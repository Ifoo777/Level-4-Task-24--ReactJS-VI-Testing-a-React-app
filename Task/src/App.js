import './App.css';
import Dictionary from './Dictionary.js';

function App() {

  return (
    <div className="App">

      <h1>API DICTIONARY</h1>

      <Dictionary/>

      <br/>
      <footer>Powered by Merriam Webster Dictionary</footer>
    </div>
  );
}

export default App;
