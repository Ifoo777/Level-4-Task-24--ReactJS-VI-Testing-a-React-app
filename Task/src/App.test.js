import React from 'react';
import App from './App';
import Renderer from 'react-test-renderer';

test('Snapshot Test',() =>{
    const tree = Renderer.create(<App/>).toJSON();
    expect(tree).toMatchSnapshot();

});