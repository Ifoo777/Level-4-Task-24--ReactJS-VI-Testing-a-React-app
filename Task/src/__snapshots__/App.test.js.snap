// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot Test 1`] = `
<div
  className="App"
>
  <h1>
    API DICTIONARY
  </h1>
  <div>
    Loading...
  </div>
  <br />
  <footer>
    Powered by Merriam Webster Dictionary
  </footer>
</div>
`;
